### ISSUE TITLE
Could not resolve "crypto" and others (trying to use it with a preprocessor )

### ISSUE THREAD

**Issue by javier-crowdar** [Zaista/cypress-mongodb#42]
I'm trying to use that with cucumber preprocessor but it doesn't recognize the commands

![image](https://github.com/Zaista/cypress-mongodb/assets/138053661/6846be3a-bb58-4659-b818-9141a9995d05)

and in terminal 
![image](https://github.com/Zaista/cypress-mongodb/assets/138053661/afc688be-b44f-44e3-b221-9ffd27f489ec)

i have this in my e2e config file 

```
e2e: {
    reporter: require.resolve("@badeball/cypress-cucumber-preprocessor/pretty-reporter"),
    // We've imported your old cypress plugins here.
    // You may want to clean this up later by importing these.
    setupNodeEvents( cypressOn, config ) {
         const on = require('cypress-on-fix')(cypressOn);

      // This is required for the preprocessor to be able to generate JSON reports after each run, and more,
      addCucumberPreprocessorPlugin(on, config);

      on(
        "file:preprocessor",
        createBundler({
          plugins: [createEsbuildPlugin(config)],
        })
      );
      configurePlugin(on)

      return config;

    }
```

any way to use it? thx)

**Comment by Zaista** [comment]
I managed to reproduce it, somehow because of the preprocessor the plugin does not get loaded, but still not sure how to fix it. Under investigation

**Comment by Zaista** [comment]
I just released the `v6.0.0` that should fix this issue.

Please make sure to update your import code in `e2e.js` file to the following:

```
import {addCommands} from 'cypress-mongodb/dist/index-browser';
addCommands();
```

And please let me know if this fixed your issue, I'd appreciate it

**Comment by javier-crowdar** [comment]
working :)
